content/support-matrix/nbp/_index.md

```diff
@@ -2,10 +2,10 @@
 title: NBP
 menuTitle: NBP
 description: "All user guides for the North Bound Plugin(NBP) project can be found here"
-weight: 10
+weight: 15
 disableToc: false
-tags: ["user guide", "delfin", "dashboard"] 
+tags: ["user guide", "nbp", "dashboard"]
```

Comment:
could you please remove the dashboard tag from here.

Reply:
Done